Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -284,7 +357,102 @@ public class SoapData
 
         private void IsSelected_FieldSelection_Click(object sender, RoutedEventArgs e)
         {
-            ObservableList<ApplicationAPIModel> CheckedList = GingerCore.General.ConvertListToObservableList(AddAPIModelWizard.AAMList.Where(x => x.IsSelected == true).ToList());
-        }        
+            ObservableList<ApplicationAPIModel> CheckedList = GingerCore.General.ConvertListToObservableList(AddAPIModelWizard.LearnedAPIModelsList.Where(x => x.IsSelected == true).ToList());
+        }
+
+        private void xCompareAndMergeButton_Click(object sender, RoutedEventArgs e)
+        {
+            ShowMergerWindow(sender);
+        }
+
+        void ShowMergerWindow(object sender)
+        {
+            DeltaAPIModel deltaAPI = null;
+            var fEl = sender as FrameworkElement;
+            if (fEl != null)
+                deltaAPI = fEl.DataContext as DeltaAPIModel;
+            if (deltaAPI != null)
+            {
+                bool showMergerWindow = false;
+                if (deltaAPI.DefaultOperationEnum == DeltaAPIModel.eHandlingOperations.MergeChanges)
+                    showMergerWindow = true;
+
+                if (mergerWindow == null)
+                    mergerWindow = new MergerWindow(deltaAPI);
+
+                mergerWindow.ownerWindow = (Window)AddAPIModelWizard.mWizardWindow;
+                mergerWindow.ShowAsWindow(showMergerWindow);
+            }
+        }
+
+        private void XHandlingOperation_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            ComboBox handlingOpCB = sender as ComboBox;
+            string mergerDescription = Convert.ToString(DeltaAPIModel.GetEnumDescription(DeltaAPIModel.eHandlingOperations.MergeChanges));
+
+            DeltaAPIModel.eHandlingOperations selectedOperation = DeltaAPIModel.GetValueFromDescription<DeltaAPIModel.eHandlingOperations>(handlingOpCB.SelectedItem.ToString());
+
+            DeltaAPIModel deltaAPI = null;
```

Comment:
why this manual binding is needed? this is how we do it in other pages where we have lists in Grids?